macros/edr/dbt_artifacts/upload_dbt_columns.sql

```diff
@@ -35,7 +35,10 @@
 
     {% set flattened_columns = [] %}
     {% for column_node in column_nodes.values() %}
-        {% do flattened_columns.append(elementary.flatten_column(table_node, column_node)) %}
+        {% set flat_column = elementary.flatten_column(table_node, column_node) %}
+        {% if (not elementary.get_config_var('upload_only_columns_with_descriptions')) or (flat_column['description'] != none and flat_column['description'] != '') %}
```

Comment:
Maybe on the right it can just be `or flat_column['description']`?

Reply:
yeah wasn't sure if it works like in python